README.md

```diff
@@ -1,20 +1,30 @@
-# Lostrómos
+# Lostromos
 
 [![Build Status](https://travis-ci.org/wpengine/lostromos.svg?branch=master)](https://travis-ci.org/wpengine/lostromos)
 [![codecov](https://codecov.io/gh/wpengine/lostromos/branch/master/graph/badge.svg)](https://codecov.io/gh/wpengine/lostromos)
 
 **NOTE**: Under active development. Not ready for production usage.
 
-Lostromos is a service that creates Kubernetes resources based on a Custom Resource
-endpoint in the Kubernetes API. It is an implementation of the [Operator
-pattern](https://coreos.com/blog/introducing-operators.html) established by CoreOS.
+## Problem statement
 
-It creates resources using Go templates, using each Custom Resource as the values
-to use during templating. It also watches the CR endpoint for creates, updates,
-and deletes and reconciles the corresponding k8s resources.
+Lostromos is a way to manage deployments through Custom Resources. This allows a user to harness the power and
+flexibility of the Kubernetes platform to more easily control resources through outside applications.
 
-It is intended to be deployed into a Kubernetes cluster. Its main configuration
-details are:
+WP Engine uses Lostromos to customize deployments into GKE. As we spin up new clusters, we use another tool to monitor
+the google api for changes and update Custom Resources as they happen. Lostromos watches for Custom Resources and
+applies a predefined template based on the information received. This allows us to deconstruct a larger service by
+deliniating based on functionality. Our applications now manage resources, and Lostromos handles the deployments.
+
+## How it works
+
+Lostromos is a service that creates Kubernetes resources based on a Custom Resource endpoint in the Kubernetes API. It
+is an implementation of the [Operator pattern](https://coreos.com/blog/introducing-operators.html) established by
+CoreOS.
+
+Lostromos manages pods via Helm or Go templates, using values from Custom Resources as new events are captured from the
```

Comment:
I think we should make this a bit more broad. Lostromos can actually create anything that you can create with Helm or Kubernetes. 

Also, did you mean "via Helm *and* Go templates"?

Reply:
So I both agree and disagree with this. We do allow for both Helm AND Go templates, but not at the same time. If you pass in a helm chart we only do helm, and if you don't we do Go. You aren't allowed to do both at the same time currently. To me both forms are correct, but OR is more correct since that is what the application allows for. It implies the ability for both Helm and Go, while specifying that they can't both be used at the same time.